FirebaseAppCheck/Tests/Unit/DeviceCheckProvider/FIRDeviceCheckAPIServiceTests.m

```diff
@@ -84,7 +84,8 @@ - (void)testAppCheckTokenSuccess {
 
   id HTTPBodyValidationArg = [self HTTPBodyValidationArgWithDeviceToken:deviceTokenData];
 
-  NSData *responseBody = [FIRFixtureLoader loadFixtureNamed:@"DeviceCheckResponseSuccess.json"];
+  NSData *responseBody =
+      [FIRFixtureLoader loadFixtureNamed:@"AttestationTokenResponseSuccess.json"];
```

Comment:
The name of the fixture file doesn't look correct.

Reply:
I guess it could be renamed to something more generic like `FACTokenExchangeResponseSuccess`.

Reply:
Yes I was wondering if i should make it more generic after using it in `AppAttestAPIServiceTests` since the contained json includes a `token` and `ttl` field. I named it `AttestationTokenResponseSuccess` to model the `AttestationTokenResponse` response message from backend.

Reply:
`FACTokenExchangeResponseSuccess` sounds good to me if you prefer that. `DeviceCheckResponseSuccess` is also fine if that's preferred. I just thought a more generic name might be ok since we use it in several testing places for different reasons.

Reply:
Yeah, good point. It looked a bit confusing because we don't really use `attestation` term in the SDK codebase

Reply:
SGTM, renaming to `FACTokenExchangeResponseSuccess`.